src/sentry/api/serializers/models/project.py

```diff
@@ -78,6 +78,8 @@
     "first-event-severity-calculation",
     "alert-filters",
     "servicehooks",
+    "similarity-embeddings-delete-by-hash",
+    "projects:similarity-embeddings-backfill",
```

Comment:
i think the projects prefix should be removed, so 
```suggestion
    "similarity-embeddings-backfill",
```

Reply:
but, if we're deleting the flag entirely, do we even need this?

Reply:
thanks! this is the backfill flag, which I can cleanup/remove in a separate PR (it's still being used though I think it's always true)

Reply:
ah i see.